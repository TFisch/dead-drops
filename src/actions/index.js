export const getProfileImage = (userInfo) => ({
  type: 'GET_IMAGE',
  userInfo
});

export const getToken = (token) => ({
  type: 'GET_TOKEN',
  token
});