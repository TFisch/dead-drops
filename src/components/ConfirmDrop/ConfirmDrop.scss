.confirm-form{
  border: 1px solid black;
  margin-left: 3em;
  margin-top: 10em;
  width: 30em;
}

.location-display{
  width: 13em;
}

.data-wrap{
  align-items: flex-start;
  display: flex;
  flex-direction: column;
  margin-left: 1em;
  margin-top: 1em;
}