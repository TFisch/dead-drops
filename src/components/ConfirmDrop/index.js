import React from 'react'

export const ConfirmDrop = () => {

  return (
    <div>
      <img className='map-icon' src={this.state.fetchedLocation} />
    </div>
  )
}
