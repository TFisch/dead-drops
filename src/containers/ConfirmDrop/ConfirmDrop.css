.button-wrap-confirm {
  display: flex;
  flex-direction: column;
  justify-content: space-between;
  padding-bottom: 3em;
  padding-left: .4em; }

.category-confirm {
  color: #86C232;
  text-shadow: 2px 1.3px black;
  text-align: center; }

.confirm-button, .edit-button {
  color: #61892f;
  font-size: 1.75em;
  text-align: center;
  text-shadow: 2px 1.3px black;
  background: none;
  border: none;
  font-weight: 700;
  padding-top: 1.75em; }
  .confirm-button:hover, .edit-button:hover {
    color: #86C232;
    cursor: pointer; }
  .confirm-button:focus, .edit-button:focus {
    color: #86C232;
    outline: none; }

.confirm-form {
  background-color: #6B6E70;
  border: 2.5px solid #86C232;
  box-shadow: 0 4px 8px 0 rgba(0, 0, 0, 0.3), 0 6px 20px 0 rgba(0, 0, 0, 0.25);
  height: 23.3em;
  margin-left: 3.2em;
  margin-top: 4.6em;
  padding: 1em;
  text-align: right;
  width: 30em; }

.column-left-wrap {
  display: flex;
  flex-direction: column;
  margin-top: .4em;
  width: 50%; }

.column-right-wrap {
  display: flex;
  flex-direction: column;
  height: 100%;
  margin-left: 1.75em;
  text-align: left;
  width: 50%; }

.data-wrap {
  display: flex;
  height: 90%;
  justify-content: space-between;
  width: 30em; }

.location-display {
  border: 2px solid black;
  box-shadow: 0 4px 8px 0 rgba(0, 0, 0, 0.3), 0 6px 20px 0 rgba(0, 0, 0, 0.25);
  margin-top: .64em;
  width: 13em; }

.output {
  color: #86C232;
  text-align: center;
  text-shadow: 1.5px 1px black; }

.stat-wrap {
  align-items: center;
  background-color: #474B4F;
  border: 2px solid black;
  box-shadow: 0 4px 8px 0 rgba(0, 0, 0, 0.3), 0 6px 20px 0 rgba(0, 0, 0, 0.25);
  height: 12em;
  margin-top: .25em;
  margin-bottom: .5em;
  width: 15em; }
