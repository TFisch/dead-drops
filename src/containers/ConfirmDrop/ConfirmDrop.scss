@import '../../styles/mixins.scss';
@import '../../styles/variables.scss';


.category-confirm {
  @include category();
  text-align: center;
}

.confirm-button {
  color: $color-two;
  font-size: 1.2em;
  text-shadow: 2px 1.3px black;

  @include underline-effect();
  &:hover{
    color: $color-one;
  }
  &:focus{
    color: $color-one;
    outline: none;
  }
}

.confirm-form {
  background-color: $color-three;
  border: 2.5px solid $color-one;
  box-shadow: $box-shadow;
  height: 23.3em;
  margin-left: 3.2em;
  margin-top: 4.6em;
  padding: 1em;
  text-align: left;
  width: 30em;
}

.column-left-wrap {
  display: flex;
  flex-direction: column;
  justify-content: flex-start;
  margin-top: 1.5em;
  width: 50%;
}

.column-right-wrap {
  display: flex;
  flex-direction: column;
  justify-content: flex-start;
  text-align: left;
  width: 50%;
}

.data-wrap {
  display: flex;
  height: 90%;
  justify-content: space-between;
}
       
.location-display {
  border: 2px solid black;
  box-shadow: $box-shadow;
  width: 13em;
}

.longitude {
  margin-top: 0;
}

.output {
  color: $color-one;
  text-align: center;
  text-shadow: 1.5px 1px black;
}

.stat-wrap {
  background-color: $color-four;
  border: 2px solid black;
  box-shadow: $box-shadow;
  margin-top: 1em;
}