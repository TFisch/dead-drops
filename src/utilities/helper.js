export const cleanImage = (imageResponse) => {
  const image = imageResponse.data.image;
  const username = imageResponse.data.username
  console.log(username);
  return image['60x60'].url;
}

