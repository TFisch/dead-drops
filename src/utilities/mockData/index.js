export const tokenMock = 'AotyU8WH1d1jf6hbf_a5y4HeZ-StFVPaN5kRpsdFOEBtoWBGwAn_ADAAAabaRTpF9OxgaLYAAAAA'

export const userReducerMock = { username: 'clark', image: 'https://via.placeholder.com/350x150' }

export const mockUsername = "Harvey";

export const mockImage = 'https://via.placeholder.com/350x150'; 