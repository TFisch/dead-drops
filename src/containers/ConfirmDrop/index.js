import React, { Component } from 'react';
import { connect } from 'react-redux';
import { postPin } from '../../api';
import './ConfirmDrop.css';
import PropTypes from 'prop-types';

export class ConfirmDrop extends Component {
  constructor(props) {
    super(props);
  }

  handleConfirm = () => {
    const { latitude, longitude, image } = this.props.location;
    const note = `Latitude: ${latitude}, Longitude: ${longitude}`;
    postPin(note, this.props.token, image);
    this.props.toggleSubmit();
  }

  handleEdit = () => {
    this.props.toggleSubmit();
  }

  render() {
    const { latitude, longitude, image, verificationCode } = this.props.location;
    return (
      <div className="confirm-form">
        <div className="data-wrap">
          <div className="column-left-wrap">
            <div className="stat-wrap">
              <h3 className="category-confirm" id="longitude">Longitude</h3>
              <h2 className="output">{longitude}</h2>
            </div>
            <div className="stat-wrap">
              <h3 className="category-confirm">Latitude</h3>
              <h2 className="output"> {latitude}</h2>
            </div>
            <div className="stat-wrap">
              <h3 className="category-confirm">Unique Verification Code</h3>
              <h2 className="output"> {verificationCode}</h2>
            </div>
          </div>
          <div className="column-right-wrap">
            <img className="location-display" src={image} alt="coordinate-location" />
            <div className="button-wrap-confirm">
              <button className="edit-button" name="edit" onClick={() => this.handleEdit()}>EDIT</button>
              <button className="confirm-button" name="confirm" onClick={() => this.handleConfirm()}>CONFIRM</button>
            </div>
          </div>
        </div>
      </div>
    );
  }
}

ConfirmDrop.propTypes = {
  token: PropTypes.string,
  toggleSubmit: PropTypes.func,
  location: PropTypes.object
};

export const mapStateToProps = (state) => ({
  token: state.token,
  location: state.locationData
});

export default connect(mapStateToProps)(ConfirmDrop);
