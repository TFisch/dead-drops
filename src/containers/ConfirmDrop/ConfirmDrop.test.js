import React from 'react';
import { ConfirmDrop } from '../ConfirmDrop';
import { shallow } from 'enzyme';
import { mockLocation, mockPropsMethod, tokenMock } from '../../utilities/mockData';
import { mapStateToProps, mapDispatchToProps } from './';
import { setFormActive } from '../../actions';

describe('ConfirmDrop', () => {
  let wrapper;
  const mockPropsMethodTwo = mockPropsMethod;
  const mockPropsMethodThree = mockPropsMethod;
  const mockPropsMethodFour = mockPropsMethod;

  beforeEach(() => {
    wrapper = shallow(<ConfirmDrop
      location={mockLocation}
      toggleSubmit={mockPropsMethod}
      postPin={mockPropsMethodTwo}
      handleEdit={mockPropsMethodFour}
      setFormActive={mockPropsMethod}
    />);
  });

  it('should match the snapshot', () => {
    expect(wrapper).toMatchSnapshot();
  });

  describe('handleConfirm', () => {

    it('should call handleConfirm on click', () => {
      wrapper.find('[name="confirm"]').simulate('click');

      expect(mockPropsMethodThree).toHaveBeenCalled();
    });

    it('should call toggleSubmit', () => {

      wrapper.instance().handleConfirm();

      expect(mockPropsMethod).toHaveBeenCalled();

    });

    it('should call the PostPins action'), () => {

      wrapper.instance().handleConfirm();

      expect(mockPropsMethodTwo).toHaveBeenCalled();

    };
  });


  describe('handleEdit', () => {

    it('should call handleEdit on click', () => {
      wrapper.find('[name="edit"]').simulate('click');

      expect(mockPropsMethodFour).toHaveBeenCalled();
    });

    it('should call toggleSubmit', () => {

      wrapper.instance().handleEdit();

      expect(mockPropsMethod).toHaveBeenCalled();

    });
  });

  describe('mapStatetoProps', () => {

    it('should return a token object', () => {
      const mockState = { token: tokenMock, locationData: {} };
      const expected = { token: tokenMock, location: {} };

      const mappedProps = mapStateToProps(mockState);

      expect(mappedProps).toEqual(expected);
    });
  });

  describe('MapDispatchToProps', () => {

    it('should call dispatch with a status value when setFormActive is called', () => {
      const mockDispatch = jest.fn();
      const mockStatus = true;
      const actionToDispatch = setFormActive(mockStatus);

      const mappedProps = mapDispatchToProps(mockDispatch);
      mappedProps.setFormActive(mockStatus);

      expect(mockDispatch).toHaveBeenCalledWith(actionToDispatch);
    });
  });

});