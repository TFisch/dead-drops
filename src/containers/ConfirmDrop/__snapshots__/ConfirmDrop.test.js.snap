// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`ConfirmDrop should match the snapshot 1`] = `
ShallowWrapper {
  Symbol(enzyme.__root__): [Circular],
  Symbol(enzyme.__unrendered__): <ConfirmDrop
    location={
        Object {
            "lat": "2.52343245",
            "locationImage": "https://via.placeholder.com/350x150",
            "long": "3214321504",
            "verificationCode": Array [
              0,
              0,
              0,
              0,
              0,
              0,
            ],
          }
    }
    postPin={[Function]}
    toggleSubmit={[Function]}
/>,
  Symbol(enzyme.__renderer__): Object {
    "batchedUpdates": [Function],
    "getNode": [Function],
    "render": [Function],
    "simulateError": [Function],
    "simulateEvent": [Function],
    "unmount": [Function],
  },
  Symbol(enzyme.__node__): Object {
    "instance": null,
    "key": undefined,
    "nodeType": "host",
    "props": Object {
      "children": <div
        className="data-wrap"
>
        <div
                className="column-left-wrap"
        >
                <div
                        className="stat-wrap"
                >
                        <h3
                                className="category-confirm"
                                id="longitude"
                        >
                                Longitude
                        </h3>
                        <h2
                                className="output"
                        />
                </div>
                <div
                        className="stat-wrap"
                >
                        <h3
                                className="category-confirm"
                        >
                                Latitude
                        </h3>
                        <h2
                                className="output"
                        >
                                 
                        </h2>
                </div>
                <div
                        className="stat-wrap"
                >
                        <h3
                                className="category-confirm"
                        >
                                Unique Verification Code
                        </h3>
                        <h2
                                className="output"
                        >
                                 
                                0
                                0
                                0
                                0
                                0
                                0
                        </h2>
                </div>
        </div>
        <div
                className="column-right-wrap"
        >
                <img
                        alt="coordinate-location"
                        className="location-display"
                        src={undefined}
                />
                <div
                        className="button-wrap-confirm"
                >
                        <button
                                className="edit-button"
                                onClick={[Function]}
                        >
                                EDIT
                        </button>
                        <button
                                className="confirm-button"
                                onClick={[Function]}
                        >
                                CONFIRM
                        </button>
                </div>
        </div>
</div>,
      "className": "confirm-form",
    },
    "ref": null,
    "rendered": Object {
      "instance": null,
      "key": undefined,
      "nodeType": "host",
      "props": Object {
        "children": Array [
          <div
            className="column-left-wrap"
>
            <div
                        className="stat-wrap"
            >
                        <h3
                                    className="category-confirm"
                                    id="longitude"
                        >
                                    Longitude
                        </h3>
                        <h2
                                    className="output"
                        />
            </div>
            <div
                        className="stat-wrap"
            >
                        <h3
                                    className="category-confirm"
                        >
                                    Latitude
                        </h3>
                        <h2
                                    className="output"
                        >
                                     
                        </h2>
            </div>
            <div
                        className="stat-wrap"
            >
                        <h3
                                    className="category-confirm"
                        >
                                    Unique Verification Code
                        </h3>
                        <h2
                                    className="output"
                        >
                                     
                                    0
                                    0
                                    0
                                    0
                                    0
                                    0
                        </h2>
            </div>
</div>,
          <div
            className="column-right-wrap"
>
            <img
                        alt="coordinate-location"
                        className="location-display"
                        src={undefined}
            />
            <div
                        className="button-wrap-confirm"
            >
                        <button
                                    className="edit-button"
                                    onClick={[Function]}
                        >
                                    EDIT
                        </button>
                        <button
                                    className="confirm-button"
                                    onClick={[Function]}
                        >
                                    CONFIRM
                        </button>
            </div>
</div>,
        ],
        "className": "data-wrap",
      },
      "ref": null,
      "rendered": Array [
        Object {
          "instance": null,
          "key": undefined,
          "nodeType": "host",
          "props": Object {
            "children": Array [
              <div
                className="stat-wrap"
>
                <h3
                                className="category-confirm"
                                id="longitude"
                >
                                Longitude
                </h3>
                <h2
                                className="output"
                />
</div>,
              <div
                className="stat-wrap"
>
                <h3
                                className="category-confirm"
                >
                                Latitude
                </h3>
                <h2
                                className="output"
                >
                                 
                </h2>
</div>,
              <div
                className="stat-wrap"
>
                <h3
                                className="category-confirm"
                >
                                Unique Verification Code
                </h3>
                <h2
                                className="output"
                >
                                 
                                0
                                0
                                0
                                0
                                0
                                0
                </h2>
</div>,
            ],
            "className": "column-left-wrap",
          },
          "ref": null,
          "rendered": Array [
            Object {
              "instance": null,
              "key": undefined,
              "nodeType": "host",
              "props": Object {
                "children": Array [
                  <h3
                    className="category-confirm"
                    id="longitude"
>
                    Longitude
</h3>,
                  <h2
                    className="output"
/>,
                ],
                "className": "stat-wrap",
              },
              "ref": null,
              "rendered": Array [
                Object {
                  "instance": null,
                  "key": undefined,
                  "nodeType": "host",
                  "props": Object {
                    "children": "Longitude",
                    "className": "category-confirm",
                    "id": "longitude",
                  },
                  "ref": null,
                  "rendered": "Longitude",
                  "type": "h3",
                },
                Object {
                  "instance": null,
                  "key": undefined,
                  "nodeType": "host",
                  "props": Object {
                    "children": undefined,
                    "className": "output",
                  },
                  "ref": null,
                  "rendered": null,
                  "type": "h2",
                },
              ],
              "type": "div",
            },
            Object {
              "instance": null,
              "key": undefined,
              "nodeType": "host",
              "props": Object {
                "children": Array [
                  <h3
                    className="category-confirm"
>
                    Latitude
</h3>,
                  <h2
                    className="output"
>
                     
</h2>,
                ],
                "className": "stat-wrap",
              },
              "ref": null,
              "rendered": Array [
                Object {
                  "instance": null,
                  "key": undefined,
                  "nodeType": "host",
                  "props": Object {
                    "children": "Latitude",
                    "className": "category-confirm",
                  },
                  "ref": null,
                  "rendered": "Latitude",
                  "type": "h3",
                },
                Object {
                  "instance": null,
                  "key": undefined,
                  "nodeType": "host",
                  "props": Object {
                    "children": Array [
                      " ",
                      undefined,
                    ],
                    "className": "output",
                  },
                  "ref": null,
                  "rendered": Array [
                    " ",
                    undefined,
                  ],
                  "type": "h2",
                },
              ],
              "type": "div",
            },
            Object {
              "instance": null,
              "key": undefined,
              "nodeType": "host",
              "props": Object {
                "children": Array [
                  <h3
                    className="category-confirm"
>
                    Unique Verification Code
</h3>,
                  <h2
                    className="output"
>
                     
                    0
                    0
                    0
                    0
                    0
                    0
</h2>,
                ],
                "className": "stat-wrap",
              },
              "ref": null,
              "rendered": Array [
                Object {
                  "instance": null,
                  "key": undefined,
                  "nodeType": "host",
                  "props": Object {
                    "children": "Unique Verification Code",
                    "className": "category-confirm",
                  },
                  "ref": null,
                  "rendered": "Unique Verification Code",
                  "type": "h3",
                },
                Object {
                  "instance": null,
                  "key": undefined,
                  "nodeType": "host",
                  "props": Object {
                    "children": Array [
                      " ",
                      Array [
                        0,
                        0,
                        0,
                        0,
                        0,
                        0,
                      ],
                    ],
                    "className": "output",
                  },
                  "ref": null,
                  "rendered": Array [
                    " ",
                    0,
                    0,
                    0,
                    0,
                    0,
                    0,
                  ],
                  "type": "h2",
                },
              ],
              "type": "div",
            },
          ],
          "type": "div",
        },
        Object {
          "instance": null,
          "key": undefined,
          "nodeType": "host",
          "props": Object {
            "children": Array [
              <img
                alt="coordinate-location"
                className="location-display"
                src={undefined}
/>,
              <div
                className="button-wrap-confirm"
>
                <button
                                className="edit-button"
                                onClick={[Function]}
                >
                                EDIT
                </button>
                <button
                                className="confirm-button"
                                onClick={[Function]}
                >
                                CONFIRM
                </button>
</div>,
            ],
            "className": "column-right-wrap",
          },
          "ref": null,
          "rendered": Array [
            Object {
              "instance": null,
              "key": undefined,
              "nodeType": "host",
              "props": Object {
                "alt": "coordinate-location",
                "className": "location-display",
                "src": undefined,
              },
              "ref": null,
              "rendered": null,
              "type": "img",
            },
            Object {
              "instance": null,
              "key": undefined,
              "nodeType": "host",
              "props": Object {
                "children": Array [
                  <button
                    className="edit-button"
                    onClick={[Function]}
>
                    EDIT
</button>,
                  <button
                    className="confirm-button"
                    onClick={[Function]}
>
                    CONFIRM
</button>,
                ],
                "className": "button-wrap-confirm",
              },
              "ref": null,
              "rendered": Array [
                Object {
                  "instance": null,
                  "key": undefined,
                  "nodeType": "host",
                  "props": Object {
                    "children": "EDIT",
                    "className": "edit-button",
                    "onClick": [Function],
                  },
                  "ref": null,
                  "rendered": "EDIT",
                  "type": "button",
                },
                Object {
                  "instance": null,
                  "key": undefined,
                  "nodeType": "host",
                  "props": Object {
                    "children": "CONFIRM",
                    "className": "confirm-button",
                    "onClick": [Function],
                  },
                  "ref": null,
                  "rendered": "CONFIRM",
                  "type": "button",
                },
              ],
              "type": "div",
            },
          ],
          "type": "div",
        },
      ],
      "type": "div",
    },
    "type": "div",
  },
  Symbol(enzyme.__nodes__): Array [
    Object {
      "instance": null,
      "key": undefined,
      "nodeType": "host",
      "props": Object {
        "children": <div
          className="data-wrap"
>
          <div
                    className="column-left-wrap"
          >
                    <div
                              className="stat-wrap"
                    >
                              <h3
                                        className="category-confirm"
                                        id="longitude"
                              >
                                        Longitude
                              </h3>
                              <h2
                                        className="output"
                              />
                    </div>
                    <div
                              className="stat-wrap"
                    >
                              <h3
                                        className="category-confirm"
                              >
                                        Latitude
                              </h3>
                              <h2
                                        className="output"
                              >
                                         
                              </h2>
                    </div>
                    <div
                              className="stat-wrap"
                    >
                              <h3
                                        className="category-confirm"
                              >
                                        Unique Verification Code
                              </h3>
                              <h2
                                        className="output"
                              >
                                         
                                        0
                                        0
                                        0
                                        0
                                        0
                                        0
                              </h2>
                    </div>
          </div>
          <div
                    className="column-right-wrap"
          >
                    <img
                              alt="coordinate-location"
                              className="location-display"
                              src={undefined}
                    />
                    <div
                              className="button-wrap-confirm"
                    >
                              <button
                                        className="edit-button"
                                        onClick={[Function]}
                              >
                                        EDIT
                              </button>
                              <button
                                        className="confirm-button"
                                        onClick={[Function]}
                              >
                                        CONFIRM
                              </button>
                    </div>
          </div>
</div>,
        "className": "confirm-form",
      },
      "ref": null,
      "rendered": Object {
        "instance": null,
        "key": undefined,
        "nodeType": "host",
        "props": Object {
          "children": Array [
            <div
              className="column-left-wrap"
>
              <div
                            className="stat-wrap"
              >
                            <h3
                                          className="category-confirm"
                                          id="longitude"
                            >
                                          Longitude
                            </h3>
                            <h2
                                          className="output"
                            />
              </div>
              <div
                            className="stat-wrap"
              >
                            <h3
                                          className="category-confirm"
                            >
                                          Latitude
                            </h3>
                            <h2
                                          className="output"
                            >
                                           
                            </h2>
              </div>
              <div
                            className="stat-wrap"
              >
                            <h3
                                          className="category-confirm"
                            >
                                          Unique Verification Code
                            </h3>
                            <h2
                                          className="output"
                            >
                                           
                                          0
                                          0
                                          0
                                          0
                                          0
                                          0
                            </h2>
              </div>
</div>,
            <div
              className="column-right-wrap"
>
              <img
                            alt="coordinate-location"
                            className="location-display"
                            src={undefined}
              />
              <div
                            className="button-wrap-confirm"
              >
                            <button
                                          className="edit-button"
                                          onClick={[Function]}
                            >
                                          EDIT
                            </button>
                            <button
                                          className="confirm-button"
                                          onClick={[Function]}
                            >
                                          CONFIRM
                            </button>
              </div>
</div>,
          ],
          "className": "data-wrap",
        },
        "ref": null,
        "rendered": Array [
          Object {
            "instance": null,
            "key": undefined,
            "nodeType": "host",
            "props": Object {
              "children": Array [
                <div
                  className="stat-wrap"
>
                  <h3
                                    className="category-confirm"
                                    id="longitude"
                  >
                                    Longitude
                  </h3>
                  <h2
                                    className="output"
                  />
</div>,
                <div
                  className="stat-wrap"
>
                  <h3
                                    className="category-confirm"
                  >
                                    Latitude
                  </h3>
                  <h2
                                    className="output"
                  >
                                     
                  </h2>
</div>,
                <div
                  className="stat-wrap"
>
                  <h3
                                    className="category-confirm"
                  >
                                    Unique Verification Code
                  </h3>
                  <h2
                                    className="output"
                  >
                                     
                                    0
                                    0
                                    0
                                    0
                                    0
                                    0
                  </h2>
</div>,
              ],
              "className": "column-left-wrap",
            },
            "ref": null,
            "rendered": Array [
              Object {
                "instance": null,
                "key": undefined,
                "nodeType": "host",
                "props": Object {
                  "children": Array [
                    <h3
                      className="category-confirm"
                      id="longitude"
>
                      Longitude
</h3>,
                    <h2
                      className="output"
/>,
                  ],
                  "className": "stat-wrap",
                },
                "ref": null,
                "rendered": Array [
                  Object {
                    "instance": null,
                    "key": undefined,
                    "nodeType": "host",
                    "props": Object {
                      "children": "Longitude",
                      "className": "category-confirm",
                      "id": "longitude",
                    },
                    "ref": null,
                    "rendered": "Longitude",
                    "type": "h3",
                  },
                  Object {
                    "instance": null,
                    "key": undefined,
                    "nodeType": "host",
                    "props": Object {
                      "children": undefined,
                      "className": "output",
                    },
                    "ref": null,
                    "rendered": null,
                    "type": "h2",
                  },
                ],
                "type": "div",
              },
              Object {
                "instance": null,
                "key": undefined,
                "nodeType": "host",
                "props": Object {
                  "children": Array [
                    <h3
                      className="category-confirm"
>
                      Latitude
</h3>,
                    <h2
                      className="output"
>
                       
</h2>,
                  ],
                  "className": "stat-wrap",
                },
                "ref": null,
                "rendered": Array [
                  Object {
                    "instance": null,
                    "key": undefined,
                    "nodeType": "host",
                    "props": Object {
                      "children": "Latitude",
                      "className": "category-confirm",
                    },
                    "ref": null,
                    "rendered": "Latitude",
                    "type": "h3",
                  },
                  Object {
                    "instance": null,
                    "key": undefined,
                    "nodeType": "host",
                    "props": Object {
                      "children": Array [
                        " ",
                        undefined,
                      ],
                      "className": "output",
                    },
                    "ref": null,
                    "rendered": Array [
                      " ",
                      undefined,
                    ],
                    "type": "h2",
                  },
                ],
                "type": "div",
              },
              Object {
                "instance": null,
                "key": undefined,
                "nodeType": "host",
                "props": Object {
                  "children": Array [
                    <h3
                      className="category-confirm"
>
                      Unique Verification Code
</h3>,
                    <h2
                      className="output"
>
                       
                      0
                      0
                      0
                      0
                      0
                      0
</h2>,
                  ],
                  "className": "stat-wrap",
                },
                "ref": null,
                "rendered": Array [
                  Object {
                    "instance": null,
                    "key": undefined,
                    "nodeType": "host",
                    "props": Object {
                      "children": "Unique Verification Code",
                      "className": "category-confirm",
                    },
                    "ref": null,
                    "rendered": "Unique Verification Code",
                    "type": "h3",
                  },
                  Object {
                    "instance": null,
                    "key": undefined,
                    "nodeType": "host",
                    "props": Object {
                      "children": Array [
                        " ",
                        Array [
                          0,
                          0,
                          0,
                          0,
                          0,
                          0,
                        ],
                      ],
                      "className": "output",
                    },
                    "ref": null,
                    "rendered": Array [
                      " ",
                      0,
                      0,
                      0,
                      0,
                      0,
                      0,
                    ],
                    "type": "h2",
                  },
                ],
                "type": "div",
              },
            ],
            "type": "div",
          },
          Object {
            "instance": null,
            "key": undefined,
            "nodeType": "host",
            "props": Object {
              "children": Array [
                <img
                  alt="coordinate-location"
                  className="location-display"
                  src={undefined}
/>,
                <div
                  className="button-wrap-confirm"
>
                  <button
                                    className="edit-button"
                                    onClick={[Function]}
                  >
                                    EDIT
                  </button>
                  <button
                                    className="confirm-button"
                                    onClick={[Function]}
                  >
                                    CONFIRM
                  </button>
</div>,
              ],
              "className": "column-right-wrap",
            },
            "ref": null,
            "rendered": Array [
              Object {
                "instance": null,
                "key": undefined,
                "nodeType": "host",
                "props": Object {
                  "alt": "coordinate-location",
                  "className": "location-display",
                  "src": undefined,
                },
                "ref": null,
                "rendered": null,
                "type": "img",
              },
              Object {
                "instance": null,
                "key": undefined,
                "nodeType": "host",
                "props": Object {
                  "children": Array [
                    <button
                      className="edit-button"
                      onClick={[Function]}
>
                      EDIT
</button>,
                    <button
                      className="confirm-button"
                      onClick={[Function]}
>
                      CONFIRM
</button>,
                  ],
                  "className": "button-wrap-confirm",
                },
                "ref": null,
                "rendered": Array [
                  Object {
                    "instance": null,
                    "key": undefined,
                    "nodeType": "host",
                    "props": Object {
                      "children": "EDIT",
                      "className": "edit-button",
                      "onClick": [Function],
                    },
                    "ref": null,
                    "rendered": "EDIT",
                    "type": "button",
                  },
                  Object {
                    "instance": null,
                    "key": undefined,
                    "nodeType": "host",
                    "props": Object {
                      "children": "CONFIRM",
                      "className": "confirm-button",
                      "onClick": [Function],
                    },
                    "ref": null,
                    "rendered": "CONFIRM",
                    "type": "button",
                  },
                ],
                "type": "div",
              },
            ],
            "type": "div",
          },
        ],
        "type": "div",
      },
      "type": "div",
    },
  ],
  Symbol(enzyme.__options__): Object {
    "adapter": ReactSixteenAdapter {
      "options": Object {
        "enableComponentDidUpdateOnSetState": true,
        "lifecycles": Object {
          "componentDidUpdate": Object {
            "onSetState": true,
          },
          "getDerivedStateFromProps": true,
          "getSnapshotBeforeUpdate": true,
          "setState": Object {
            "skipsComponentDidUpdateOnNullish": true,
          },
        },
      },
    },
    "attachTo": undefined,
    "disableLifecycleMethods": true,
    "hydrateIn": undefined,
  },
}
`;
